src/DropDownMenu/DropDownMenu.spec.js

```diff
@@ -31,6 +32,43 @@ describe('<DropDownMenu />', () => {
     });
   });
 
+  describe('prop: disabled', () => {
+    it('disables the touchRipple effect if disableTouchRipple is undefined and disabled={true}', () => {
+      const wrapper = mountWithContext(
```

Comment:
Would you say that the unit tests for SelectField are extraneous then? Because the behavior is dependent on DropDownMenu